saos-search/src/main/java/pl/edu/icm/saos/search/indexing/ReindexJobStepExecutionListener.java

```diff
@@ -35,6 +44,14 @@
     public void beforeStep(StepExecution stepExecution) {
         SourceCode sc = EnumUtils.getEnum(SourceCode.class, sourceCode);
         judgmentRepository.markAsNotIndexedBySourceCode(sc);
+        
+        if (sc == null) {
```

Comment:
As to the first part - yes, I think we can accept it.

As to the latter one, it's a good question. Now courtType corresponds directly to SourceCode, but it does not have to in the future. SourceCode tells us where the given judgment has come from and courtType tells what type of judgment it is. Now, the reindexing of only a part of  judgments is most often caused by a change in an import from a given source, so I guess the perfect solution would be adding the sourceCode to the index.